satpy/readers/avhrr_l1b_gaclac.py

```diff
@@ -299,6 +310,18 @@ def _strip_invalid_lat(self):
             self.first_valid_lat, self.last_valid_lat = start, end
         return self.first_valid_lat, self.last_valid_lat
 
+    def _update_attrs(self, res):
+        """Update dataset attributes."""
+        for attr in self.reader.meta_data:
+            res.attrs[attr] = self.reader.meta_data[attr]
+        res.attrs['platform_name'] = self.reader.spacecraft_name
+        res.attrs['orbit_number'] = self.filename_info['orbit_number']
```

Comment:
what if the filepattern doesn't provide an orbit ? Should we have some default value ?

Reply:
@mraspaud would you have an example for that?

Reply:
I'm just thinking of the case where the user has rename its file and there is no orbit number anymore.
so something like `res.attrs['orbit_number'] = self.filename_info.get('orbit_number', None)` ?

Reply:
I had something different in mind - thanks :)